src/mumble/MumbleApplication.cpp

```diff
@@ -56,46 +60,47 @@ bool MumbleApplication::event(QEvent *e) {
 }
 
 #ifdef Q_OS_WIN
+/// gswForward forwards a native Windows keyboard/mouse message
+/// into GlobalShortcutWin's event stream.
+static void gswForward(MSG *msg) {
+	if (Global::g_global_struct) {
```

Comment:
I had problems with this code in the crash reporter at some point. I'll investigate.